docs/src/options/definitions.md

```diff
@@ -185,11 +185,6 @@
 - Type: boolean
 - Default: "true"
 
-## solution\_file
```

Comment:
In case you're wondering about this deletion: this option is documented below (line 230)